test/helpers.ts

```diff
@@ -0,0 +1,19 @@
+import * as sinon from 'sinon';
+
+function stubRequest(): any  {
```

Comment:
Could you just return type Request, if you're returning it as a Request?

Reply:
The problem is Request doesn't have any of the stub related functions, so has to be recast before it can be used by `expect(...)`.